### ISSUE TITLE
Adding febio fluid simulation demo to gibbon library

### ISSUE THREAD

**Issue by muntahi398** [gibbonCode/GIBBON#97]
GIbbon has multiple demo for FEA analysis using FEBIO. Adding some CFD demo would be very much appreciated.

**Comment by Kevin-Mattheus-Moerman** [comment]
Yes, good point! I have some FSI codes that could form the basis of this. I'll work on this the coming months and will let you once it is online. In the meantime feel free to post questions here on this.

**Comment by Kevin-Mattheus-Moerman** [comment]
Good news we are currently formulating such as demo! I will let you know when it is ready .

**Comment by muntahi398** [comment]
Thank you very much. Could you show one using an stl just like [Link1](https://seafile.zfn.uni-bremen.de/f/b6e34cc0c0a84d9884db/?dl=1) or [Link2](https://seafile.zfn.uni-bremen.de/f/4ea507fa90c4423c8faf/?dl=1)
to run the CFD simulatiion as you have done for the Demo 21, bone 3 point bending  simulation.

**Comment by cbcapron** [comment]
Hi, I was just curious if any such FSI demo is (or could be made) available? I am interested in using gibbon for some FEBio FSI simulations also.
